.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.3.0
```

Comment:
It should be there if you upgrade rbenv (homebrew or git) because I've been using 2.3.1 since it came out.